src/app/test-record/defects/defects.component.spec.ts

```diff
@@ -1,8 +1,8 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { DefectsComponent } from './defects.component';
 import { SharedModule } from '@app/shared/shared.module';
-import { TEST_MODEL_UTILS } from '@app/utils/test-models.utils';
 import { Defect } from '@app/models/defect';
+import { TESTING_TEST_MODELS_UTILS } from '@app/utils/testing-test-models.utils';
```

Comment:
please use `TEST_MODEL_UTILS` for your export. Filename to just be `test-model.utils`